bot/cogs/pavlov_mod.py

```diff
@@ -206,7 +206,16 @@ async def setpin(
         """
         if not await check_perm_moderator(ctx, server_name):
             return
-        data = await exec_server_command(ctx, server_name, f"SetPin {pin}")
+        if len(pin) == 4 and pin.isdigit():
+            data = await exec_server_command(ctx, server_name, f"SetPin {pin}")
+        elif pin == 'remove':
```

Comment:
When checking for `== 'remove'` I would probably lowercase the pin in case someone types `Remove` or similar.